scripts/i18n_utility/README.md

```diff
@@ -0,0 +1,106 @@
+# I18n Utility Script
+
+This is a CLI utility for:
+* identifying orphaned translation related files
+* deleting orphaned files
+* identifying rename changes to make for translation related files
+* making rename changes for translation related files
+
+Orphaned files refer to files that exist underneath `i18n/content/docs` that do not have an english equivalent and are therefore unused and safe to delete.
+
+For the rename changes, this refers to when you move an english document and don't move an existing translated version of that doc. So if we have a doc `hello/world.txt` and we rename it to `hello/goodbye.txt` we also want to rename the translated versions of that document (if they exist). The last two commands will print out subsequent rename changes that need to be made (based on what has been renamed already in the english content directory) and make those changes for you.
```

Comment:
I was briefly baffled by this paragraph. I'm sure we can clarify and simplify this explanation. Also, why are we using .txt for examples when we work entirely in .mdx files :)

"When you rename a doc's filename, you need to make sure the translated file is renamed to match.

/docs/content/../hello-world.mdx   --> /docs/content/../goodbye-world.mdx  

/docs/content/../i18n/../hello-world.mdx --> /docs/content/../i18n/../goodbye-world.mdx 
"

Or whatever the paths are

Reply:
just to nit pick, this change isn't limited to mdx files, but everything else i agree with 🙈 

any file that lives under the i18n content directory is a candidate. .txt is probably an inferior example compared to .mdx, but a fair example would be images.